src/Microsoft.Health.Fhir.TemplateManagement/Overlay/OverlayFileSystem.cs

```diff
@@ -59,36 +64,74 @@ public void WriteMergedOCIFileLayer(OCIFileLayer oneLayer)
 
         public List<OCIArtifactLayer> ReadImageLayers()
         {
-            return ReadOCIArtifactLayers(WorkingImageLayerFolder);
+            var manifest = ReadManifest();
+            var imageLayers = ReadOCIArtifactLayers(WorkingImageLayerFolder);
+            var sortedLayers = new List<OCIArtifactLayer>();
+            ValidationUtility.ValidateManifest(manifest);
+            foreach (var layerInfo in manifest.Layers)
```

Comment:
Shall we add a null-check for `manifest`? Since the return value of `ReadManifest()` might be null.
(`JsonConvert.DeserializeObject` will return null if `manifestContent` is an empty string).

Reply:
We validate manifest in ValidationUtility.ValidateManifest(manifest), exception will be thrown if manifest is null.

Reply:
Got it, thanks!